tools/spelling/spelling_dictionary.txt

```diff
@@ -261,52 +229,33 @@ RSA
 RST
 RTDS
 RTTI
-RUNDIR
-RW
```

Comment:
commonly used and understood. Please revert.

Reply:
Reverted RW